src/client/scenes/Preloader.ts

```diff
@@ -34,7 +36,71 @@ export default class Preloader extends Phaser.Scene {
     })
   }
 
-  create() {
-    this.scene.start('game')
+  async create() {
+    // create loading texts
+    const screenCenterX = this.cameras.main.worldView.x + this.cameras.main.width / 2
+    const screenCenterY = this.cameras.main.worldView.y + this.cameras.main.height / 2
+    this.add
+      .text(screenCenterX, screenCenterY - 100, 'SkyOffice')
+      .setOrigin(0.5)
+      .setFontSize(50)
+      .setFontStyle('bold')
+      .setColor('#000000')
+    const loadingText = this.add
+      .text(screenCenterX, screenCenterY - 30, 'Loading...')
+      .setOrigin(0.5)
+      .setFontSize(30)
+      .setColor('#000000')
+    this.add
+      .text(
+        screenCenterX,
+        screenCenterY + 40,
+        'Please patiently wait for 10-15 seconds because it takes some time to wake the server up.'
```

Comment:
I will change this to "It takes some time to wake up Sky, our server. Please be patient and Sky will be with you shortly."